Vostok.Metrics.System/Vostok.Metrics.System.csproj

```diff
@@ -36,5 +36,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\vostok.metrics\Vostok.Metrics\bin\Release\netstandard2.0\Vostok.Metrics.dll</HintPath>
     </Reference>
+    <Reference Include="Vostok.Sys.Metrics.PerfCounters, Version=0.0.2.0, Culture=neutral, PublicKeyToken=null">
```

Comment:
We typically reference other libraries with `cm ref add` command which produces a ref without explicit version. Please adjust this one to match other refs.

Reply:
Also, you should release a new version of `vostok.sys.metrics.perfcounters` module with recent changes to avoid stale NuGet dependencies.

Reply:
+ a comment for `HostMonitorExtensions` not changed in this PR. I suggest adding network utilization metrics to the logged summary. Something like this: `Network: in = X MB/sec; out = Y MB/sec`.